src/hooks/useKeyEvent/__stories__/useKeyEvent.mdx

```diff
@@ -124,11 +106,8 @@ Attaches a listener to keyboard DOM events on a specific element, firing a provi
       type="string"
       description={
         <>
-          The event to listen to. See{" "}
-          <Link
-            href="https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent#events" withoutSpacing>
```

Comment:
Isn't it better with the `withoutSpacing` prop? Instead of margin it adds a blank space. I wonder whether is should be the default

Reply:
This stuff was the build-killer
<img width="134" alt="image" src="https://github.com/mondaycom/monday-ui-react-core/assets/104433616/d0ed0b53-b0cf-4159-a356-628e58a24295">

Turns out that new converter they use doesn't allow such syntax -

Reply:
Oh no

Reply:
Not all `withoutSpacing`, don't worry, all good :) 
But lot's of them :D